esp/esp/users/views/usersearch.py

```diff
@@ -139,8 +135,7 @@ def get_user_list(request, listDict2, extra=''):
         keys = request.POST['keys'].split(',,')
 
         for key in keys:
-            if request.POST.has_key('operator_'+key) and \
-               request.POST['operator_'+key]         and \
+            if request.POST.get('operator_'+key) and \
```

Comment:
Oh, whoops, I misread the diff on this one, you're right.

I was continuing my thought from the other comment several files up the page! I dunno, it made sense in my head.